spec/snapshots/kitchen_spec/directions___Kitchen::Directions::BakeChapterKeyConcepts___when append_to is nil___works.snap

```diff
@@ -0,0 +1,54 @@
+<div data-type="chapter"><div data-type="page">
```

Comment:
should this file (& the ones below) also be in the `snapshots/kitchen_spec/directions` folder?

Reply:
I'm not sure why some snapshot filenames keep the `/` (causing the subdirectory) while others do not but I think it might be because the base_path from the rspec metadata may sometimes have a `/` in it.

I updated `match_snapshot_auto` to replace all slashes with `_`. It also sanitizes the filenames so characters like ` ` and `<` are converted to underscores.

This could lead to collisions. If it does, a hash could be appended to the snapshot filename (it's commented in the PR)